.lintrunner.toml

```diff
@@ -216,8 +216,10 @@ include_patterns = [
     'torch/csrc/*.cpp',
     'torch/csrc/**/*.h',
     'torch/csrc/**/*.cpp',
-    'torch/csrc/distributed/autograd/*.cpp',
-    'torch/csrc/distributed/autograd/*.h',
+    'torch/csrc/distributed/autograd/**/*.cpp',
```

Comment:
This is a no-op and should be covered by lines 217-218. Is there a bug with lintrunner that needs to be patched?

Reply:
torch/csrc/distributed was excluded in the excluded pattern. I want to gradually enable it by specifying sub-directories.